fullstop-timemachine/src/main/java/org/zalando/stups/fullstop/swagger/api/FullstopApi.java

```diff
@@ -159,6 +126,21 @@ public Violation resolveViolations(
         return mapToDto(dbViolationEntity);
     }
 
+    @ApiOperation(value = "Put instance log in S3", notes = "Add log for instance in S3", response = Void.class)
+    @ApiResponses(value = { @ApiResponse(code = 201, message = "Logs saved successfully") })
+    @RequestMapping(value = "/instance-logs", method = RequestMethod.POST)
+    @PreAuthorize("permitAll")
```

Comment:
yes, at least until the last old Taupage image, which was not able to do security, vanished.